cedar-example-use-cases/document_cloud/README.md

```diff
@@ -1,75 +1,102 @@
+
+For more information about exploring this example using the Cedar CLI, see [Cedar Example Use Cases](https://github.com/cedar-policy/cedar-examples/tree/release/4.0.x/cedar-example-use-cases).
+
+## Use-case
+
+Envision a cloud-based document sharing system, like Google Drive or Dropbox. This system can be used by a single user who is working on documents across multiple computers, by multiple users who are collaborating on a shared set of documents, or by the public as a hosting solution. Users need to be able to upload, delete, and modify the sharing permissions on their documents. Users also need to be able to view, comment on, and modify documents that they have access to. The system enforces correct access control logic. Since this is a multi-tenant system, it must have  a mechanism to protect against cross-user abuse. This system includes a blocklist feature to accomplish this.
 
-We first define the entities involved in this system.
 ## Entities
 
 ### `User`
-`User`s are the main principals of the system. They are the ones who view/edit/delete documents, as well as the ones who control sharing permissions on documents. `User`s may also block other users. For instance, if Alice blocks Bob, then Bob should not be able to view any documents Alice owns, or share anything with Alice.
+Users are the main principals of the system. They are the ones who view/edit/delete documents, as well as the ones who control sharing permissions on documents. They may also block other users. For instance, if Alice blocks Bob, then Bob should not be able to view any documents Alice owns or share anything with Alice.
 
 ### `Group`
-For convenience, `User`s can be organized into `Group`s. Documents can be shared with entire `Group`s at once. We’ll borrow from Unix and say that every `User` also has a `Group` containing only them.
+For convenience, users can be organized into groups. Documents can be shared with entire groups at once. We’ll borrow from Unix and say that every user also has a group containing only them.
 
 ### `Public`
-A principal that represents an un-authenticated user.
+A principal that represents an unauthenticated user.
 
 ### `Document`
 
-`Document`s are the core resource of the system. Every document has an owner, which is the `User` who created it. Documents have 3 axis of sharing
+Documents are the core resource of the system. Every document has an owner, which is the user who created it. Documents have 3 methods of sharing
 
 * Private: only the owner can view/edit/comment/delete the document
-* ACL: List of users/groups that are allowed view, groups allowed to comment, groups allowed to edit, groups allowed to manage.
-* Public: Can the public view/edit/comment on the document
-
-It is always enforced that only the owner can delete or edit the sharing state of a document
+* Access-control List (ACL): List of users/groups that are allowed view, groups allowed to comment, groups allowed to edit, groups allowed to manage.
```

Comment:
```suggestion
* Access-control List (ACL): List of users/groups that are allowed to view, groups allowed to comment, groups allowed to edit, groups allowed to manage.
```

Reply:
On a second thought, maybe we can avoid repeating "groups allowed to" here? I suppose ACLs are really just a way to define users/groups allowed view, comment, edit or manage documents.